src/main/java/com/google/devtools/build/lib/analysis/platform/ConstraintValueInfo.java

```diff
@@ -58,6 +62,31 @@ public Label label() {
     return label;
   }
 
+  /**
+   * Returns a {@link ConfigMatchingProvider} that matches if the owning target's platform includes
+   * this constraint.
+   *
+   * <p>The {@link com.google.devtools.build.lib.rules.platform.ConstraintValue;ConstraintValue}
```

Comment:
The `@link` syntax looks wrong here, what are you intending to do?

Reply:
Right. Just removing the part after the semi-colon works. I fully qualified it because otherwise an import on its package adds a dependency which messes with the BUILD dependency structure. Which isn't worth it just for a code link (I believe that's our actual style guidance too: to not add new dependencies just for doc links).